### PR TITLE
Add startupprobe

### PR DESCRIPTION
**Pull request by madeleine666**
*Description of changes:*
Added startupProbe value and configuration to the helm chart for kibana, es-client, es-master and es-data. Updated the documentation with this new values.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by peterzhuamazon** [comment]
Hi @madeleine666 thanks for this PR about startupProbe.
Did you already test this on your local env?
If you, could you post some logs/results for that?

Thanks.

**Comment by madeleine666** [comment]
Hi @peterzhuamazon, 
on my local env I did just a lint and tested with helm template for stupid mistakes. The deployment in our testcluster I did with argocd. Tested on: Server Version: "v1.18.9". It is not really verbose, but also no errors.

Same erros on master:
$ git checkout add-startupprobe && helm lint helm/opendistro-es/
Switched to branch 'add-startupprobe'
==> Linting helm/opendistro-es/
[INFO] Chart.yaml: icon is recommended
[ERROR] templates/elasticsearch/es-client-ingress.yaml: object name does not conform to Kubernetes naming requirements: ""
[ERROR] templates/elasticsearch/es-client-pdb.yaml: object name does not conform to Kubernetes naming requirements: ""
[ERROR] templates/elasticsearch/es-data-pdb.yaml: object name does not conform to Kubernetes naming requirements: ""
[ERROR] templates/elasticsearch/es-master-pdb.yaml: object name does not conform to Kubernetes naming requirements: ""
[ERROR] templates/elasticsearch/role.yaml: the kind "rbac.authorization.k8s.io/v1beta1 Role" is deprecated in favor of "rbac.authorization.k8s.io/v1 Role"
[ERROR] templates/kibana/kibana-config-secret.yaml: object name does not conform to Kubernetes naming requirements: ""
[ERROR] templates/kibana/role.yaml: the kind "rbac.authorization.k8s.io/v1beta1 Role" is deprecated in favor of "rbac.authorization.k8s.io/v1 Role"

Error: 1 chart(s) linted, 1 chart(s) failed
`

File: helm/opendistro-es/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ name: opendistro-es
 sources:
 - https://pages.git.viasat.com/ATG/charts
 # Chart version
-version: 1.10.1
+version: 1.10.2
```
Comment (peterzhuamazon) [inline helm/opendistro-es/Chart.yaml:29]: Hi @madeleine666 this change is not related to add startupprobe.
Please consider making a new PR for such change.
Thanks.

Comment (madeleine666) [reply helm/opendistro-es/Chart.yaml]: Hi @peterzhuamazon, 
my mistake, removed it.

**Review by peterzhuamazon — APPROVED** [review]

**Review by camerski — APPROVED** [review]

**Comment by peterzhuamazon** [comment]
PR merged. Thanks @madeleine666 for the contribution here.
